bokehjs/src/coffee/renderer/tile/mercator_tile_source.coffee

```diff
@@ -4,30 +4,25 @@ TileSource = require "./tile_source"
 class MercatorTileSource extends TileSource
 
   type: 'MercatorTileSource'
-  
-  defaults: =>
-    return _.extend {}, super(), {
-      tile_size : 256
-      max_zoom : 30
-      min_zoom : 0
-      extra_url_vars : {}
-      full_extent : [-20037508.34, -20037508.34, 20037508.34, 20037508.34]
-      x_origin_offset : 20037508.34
-      y_origin_offset : 20037508.34
-      initial_resolution : undefined
-      resolutions : undefined
-    }
-
-  constructor: ->
-    super
-
-    # compensation for not knowing how to do computed default properties
-    # warning: order matters here...
+
+  initialize: (options) ->
+
+    super(options)
+
     if not @get('initial_resolution')?
-      @set('initial_resolution', 2 * Math.PI * 6378137 / @get('tile_size'))
+      @set('initial_resolution', 2 * Math.PI * 6378137 / @get('tile_size'), {silent: true})
```

Comment:
Yes, Good point and scenario. Computing in initialize won't be run again, and it should if `tile_size` is changed.  Might be good to handle in follow up PR.  At this for current examples, `tile_size` / `initial_resolution` shouldn't change.